docs/_pages/typecheck.md

```diff
@@ -128,16 +128,25 @@ local v2 = vec2(1, 2)
 v2.z = 3 -- not ok
 ```
 
+Unsealed tables are *exact* in that any property of the table must be named by the type. Since Luau treats missing properties as having value `nil`, this means that we can treat an unsealed table which does not mention a property as if it mentioned the property, as long as that property is optional.
```

Comment:
Is "exact" the correct language here? I'd normally expect it to mean "any properties not listed in the type are absent from the table".

Reply:
Ah I think I understand what this means. Any explicit updates to the table will be recorded in the type which makes it exact, short of cases where it gets a new field in a context type checker can't understand.